.github/workflows/editor-only.yml

```diff
@@ -46,13 +46,13 @@ jobs:
 
   sign-editor-macos:
     needs: [build-editor]
-    runs-on: macOS-13
+    runs-on: macOS-14
     strategy:
       matrix:
         platform: [x86_64-macos, arm64-macos]
     steps: [
       { name: 'Checkout', uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 },
-      { name: 'Install Python', uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c, with: { python-version: 3.12, architecture: x64 } },
```

Comment:
we don't have to specify arch anymore (each mac should pick an arch that fits better)